src/routes/roadmap/roadmapData.ts

```diff
@@ -0,0 +1,77 @@
+export interface RoadmapItems {
+	icon: string;
+	status: 'completed' | 'in-progress' | 'planned' | 'milestone';
+	title: string;
+	summary: string;
+	date: string;
+}
+
+export const roadmapItems: RoadmapItems[] = [
+	{
+		icon: '💚',
+		status: 'milestone',
+		title: 'Established',
+		summary: 'Work on the Felt project begins in earnest.',
+		date: 'Q1 2020',
+	},
+	{
+		icon: '⭐️',
+		status: 'completed',
+		title: 'Applied for Start.coop accelerator',
+		summary:
+			'Our first time pitching the project. We did not get in, but the experience was invaluble.',
+		date: 'Q1 2020',
+	},
+	{
+		icon: '💻',
+		status: 'completed',
+		title: 'First mockup published',
+		summary: 'A first mockup of the UX is put together and released.',
+		date: 'Q1 2020',
+	},
+	{
+		icon: '📝',
+		status: 'completed',
+		title: 'Published "Why Svelte"',
+		summary:
+			'Published this piece (https://github.com/feltcoop/why-svelte) about why we are using Svelte.',
```

Comment:
```suggestion
			'Published the technical article <a href="https://github.com/feltcoop/why-svelte">"Why Svelte is our choice for a large web project in 2020"</a>',
```

formatting may be screwed up by this change